ci/linkcheck.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -ev
+set -o pipefail
+
+# https://docs.travis-ci.com/user/environment-variables/#default-environment-variables
+if [ "$TRAVIS_EVENT_TYPE" = "cron" ] ; then # running in cron job
+  FLAGS=""
+
+  echo "Doing full link check."
+elif [ "$CI" = "true" ] ; then # running in PR CI build
+  if [ -n "$TRAVIS_COMMIT_RANGE" ]; then
+    echo "error: unexpected state: COMMIT_RANGE must be non-empty in CI"
```

Comment:
oh oops, you meant to add a `!`. But at that point we might as well use `-z`:
```suggestion
  if [ -z "$TRAVIS_COMMIT_RANGE" ]; then
    echo "error: unexpected state: COMMIT_RANGE must be non-empty in CI"
```
```
    String operators:
    
      -z STRING      True if string is empty.
    
      -n STRING
         STRING      True if string is not empty.
```

Reply:
Oops, I left the same comment :)